lib/dot.js

```diff
@@ -12,6 +12,9 @@ export default class Dot extends TileLayer.Canvas {
   initialize (grid, color = '#49a0d7') {
     this.color = new Color(color).rgbArray()
     this.grid = grid
+    //this.colorScale = new CutoffColorScale(1000, [0, 0, 0, 0], [...this.color, 128])
+    this.colorScale = new InterpolatingColorScale([[200, 0, 0, 200, 0], [2000, 0, 0, 200, 180], [20000, 150, 0, 0, 180]])
```

Comment:
It's a leaflet layer, and leaflet uses the initialize method, I think because it was built before es6 standardized classes.